eigenda/reader.go

```diff
@@ -93,8 +93,8 @@ func ParseSequencerMsg(abiEncodedCert []byte) (*EigenDABlobInfo, error) {
 		return nil, err
 	}
 
-	// decode to EigenDABlobInfo
-	var blobInfo EigenDABlobInfo
+	// decode to EigenDAV1Cert
+	var blobInfo EigenDAV1Cert
```

Comment:
nit: var is still named "blobInfo"

Reply:
Awk - will address this in a follow up PR